eventrouter.go

```diff
@@ -78,19 +111,52 @@ func (er *EventRouter) Run(stopCh <-chan struct{}) {
 	<-stopCh
 }
 
-// addEvent is called ehen an event is created, or during the initial list
+// addEvent is called when an event is created, or during the initial list
 func (er *EventRouter) addEvent(obj interface{}) {
 	e := obj.(*v1.Event)
+	prometheusEvent(e)
 	er.eSink.UpdateEvents(e, nil)
 }
 
 // updateEvent is called any time there is an update to an existing event
 func (er *EventRouter) updateEvent(objOld interface{}, objNew interface{}) {
 	eOld := objOld.(*v1.Event)
 	eNew := objNew.(*v1.Event)
+	prometheusEvent(eNew)
 	er.eSink.UpdateEvents(eNew, eOld)
 }
 
+// prometheusEvent is called when an event is added or updated
+func prometheusEvent(event *v1.Event) {
+	var counter prometheus.Counter
+	var err error
+
+	if event.Type == "Normal" {
+		counter, err = kubernetesNormalEventCounterVec.GetMetricWithLabelValues(
+			event.InvolvedObject.Kind,
+			event.InvolvedObject.Name,
+			event.InvolvedObject.Namespace,
```

Comment:
Could be another input option, e.g.  https://github.com/heptio/eventrouter/blob/master/main.go#L69